.github/ISSUE_TEMPLATE/documentation_request.yml

```diff
@@ -0,0 +1,64 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Documentation Request
+description: Request updates or additions to cuDF's documentation
+title: "[DOC]: "
+labels: ["doc"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to help cuDF and improve our documentation!
+        - By submitting this issue, you agree to follow our [Code of Conduct](https://docs.rapids.ai/resources/conduct/)
+        - You also confirm that you have searched the [open documentation issues](https://github.com/rapidsai/cuDF/issues?q=is%3Aopen+is%3Aissue+label%3Adoc) and have found no duplicates for this request
+
+  - type: dropdown
+    id: criticality
+    attributes:
+      label: How would you describe the priority of this documentation request
+      options:
+        - Critical (currently preventing usage)
+        - Medium
+        - Low (would be nice)
```

Comment:
This is fuzzy to me -- and thus potentially a source of friction for issue filers. "Critical documentation fix" feels like a bit of a contradiction to me. Do others think this field is helpful/necessary?

Reply:
https://github.com/rapidsai/cuml/issues/5252 is an example of a critical doc issue imo, but happy to remove this dropdown if the team doesn't find value in it. 

The intention of these dropdowns is to allow the user to directly tell us how important they feel something is so we don't need to intuit it. If it costs us more than it benefits us, then that's not great and should be removed.

Reply:
My feeling is that this more likely than not to just cause annoyance. We are going to triage importance _anyway_ and if we decide that, in our estimation, a user's "critical" issue is in fact low priority what then?